web/angular-wallet/src/app/login/login-active/login-active.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+@Component({
+  selector: 'app-login-active',
```

Comment:
we have some spacing inconsistencies... here it is 2 WS, in other files it is TAB. I already applied tslint for webstorm, but still not satisfying. I prefer Tabs btw